doc/advanced-topics.md

```diff
@@ -0,0 +1,298 @@
+## Advanced Topics
```

Comment:
I understand this is a new document with content previously on README.md about advance topics.

I see the new document is linked from mkdocs.yml, but it should be also linked from README.md and I haven't found that link (maybe I missed?).

Reply:
Readme updated. Fix c45ba05